server/plugin/api_test.go

```diff
@@ -279,7 +279,8 @@ func TestHandleLink(t *testing.T) {
 				mockedStore.EXPECT().GetAllProjects("mockMattermostUserID").Return(testCase.projectList, nil)
 				mockedStore.EXPECT().StoreProject(&serializers.ProjectDetails{
 					MattermostUserID: "mockMattermostUserID",
-					OrganizationName: "mockOrganization",
+					ProjectName:      "Mockproject",
+					OrganizationName: "mockorganization",
```

Comment:
```suggestion
					ProjectName:      "mockProject",
					OrganizationName: "mockOrganization",
```

Reply:
We are making the project name tilte case and org name as smallcase here